common/test/acceptance/pages/lms/login.py

```diff
@@ -16,37 +16,41 @@ class LoginPage(PageObject):
     url = BASE_URL + "/login"
 
     def is_browser_on_page(self):
-        return any([
-            'log in' in title.lower()
-            for title in self.q(css='span.title-super').text
-        ])
+        return self.browser.title.startswith("Sign in or Register")
 
-    def login(self, email, password):
+    def login(self, email, password, expected_redirect=None):
         """
         Attempt to log in using `email` and `password`.
+        `expected_redirect` is a PageObject that corresponds to the page
+        that we expect the user to be redirected to after login.
         """
         self.provide_info(email, password)
-        self.submit()
+        self.submit(expected_redirect)
 
     def provide_info(self, email, password):
         """
         Fill in login info.
         `email` and `password` are the user's credentials.
         """
-        EmptyPromise(self.q(css='input#email').is_present, "Click ready").fulfill()
-        EmptyPromise(self.q(css='input#password').is_present, "Click ready").fulfill()
+        EmptyPromise(self.q(css='input#login-email').is_present, "Click ready").fulfill()
```

Comment:
@adampalay it looks like `LoginPage` is meant to cover the LMS' old, pre-logistration login page. That page still exists in edx-platform and is the platform's default login page, but we don't use it.

I was under the impression that the page had been kept around because it supported theming better than the logistration templates. However, that may no longer be true given recent work to better-support theming in the platform. `LoginPage` can be removed; there's only one test using it, and it's not a test meant to verify the behavior of the page. Maybe this should be the impetus for removing the old login page once and for all.